src/lib/radio/radio.ts

```diff
@@ -332,6 +332,7 @@ export const _MdRadioButtonMixinBase = mixinColor(mixinDisableRipple(MdRadioButt
     '[class.mat-radio-checked]': 'checked',
     '[class.mat-radio-disabled]': 'disabled',
     '[attr.id]': 'id',
+    '(focus)': '_inputElement.nativeElement.focus()',
```

Comment:
I think it would be really beneficial to have a comment for this. What do you think?

A comment that explains that normally the native element should be focused; But for the focus-trap there is an exception